packages/transaction_signing_gateway/lib/encrypt/aes_cipher.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:encrypt/encrypt.dart';
+import 'package:transaction_signing_gateway/encrypt/cipher.dart';
+
+class AESCipher implements Cipher {
```

Comment:
@alain2sf  can you take a look if this implementation of encrypting/decrypting data using just password makes sense?